internal/agent/agent.go

```diff
@@ -82,6 +91,17 @@ func (a *Agent) Configure() {
 	if a.rateLimit > 0 {
 		a.semaphore = patterns.NewSemaphore(a.rateLimit)
 	}
+
+	if a.secureMode {
+		publicKey, err := mycrypt.LoadPublicKey(a.publicKeyFile)
+		if err != nil {
+			logger.Log.Warn("Failed to load public key. SecureMode set False", zap.String("file", a.publicKeyFile))
+			logger.Log.Error(err.Error())
```

Comment:
Не совсем понятно почему пишется две записи в лог на уровне Warn и Error, мне кажется будет достаточно одной записи: 
```suggestion
			logger.Log.Error("Failed to load public key. SecureMode set False", zap.Error(err), zap.String("file", a.publicKeyFile))
```